### DISCUSSION TITLE
where did the server folder go in new nuxt apps

### DISCUSSION THREAD

**Discussion by DerekAlanGilbert** [nuxt/nuxt#8968] [Questions (Nuxt 2)]
My newer nuxt apps dont scaffold from create-nuxt-app with a server folder?  is it all under the hood now? where did it go

**Comment by danielroe** [comment]
Some background at https://github.com/nuxt/create-nuxt-app/pull/501 and https://github.com/nuxt/create-nuxt-app/issues/563.

**Reply by DerekAlanGilbert** [reply]
@danielroe thank you very much. Ill read through these threads.
